docs/docs/development/setup/setup.md

```diff
@@ -0,0 +1,19 @@
+---
+title: General Info
+sidebar_label: General Information
+---
+
+## Environment Setup
+
+There are 2 ways to setup the zmk development environment: Docker+VSCode and Native OS. The Docker+VSCode approach is a self-contained development environment while Native OS approach will setup your local operating system for development.
+
+The Docker+VSCode approach is very straight forward to setup. The Native approach requires more up-front work but doesn't require installing Docker/VSCode and can be faster for development depending on your computer's hardware.
+
+The links in the navigation bar will outline the necessary steps for setting up both of these approaches.
+
+:::danger The Docker+VSCode environment will **NOT** run on arm CPUs. You must use the Native OS environment if using an arm CPU.
+:::
+
+## ZMK Sources Setup
+
+Once you've setup either the Docker+VSCode environment or Native OS environment you will need to setup the ZMK sources. Please see the [ZMK Sources](/docs/development/setup/zmk) section once you've setup your chosen environment.
```

Comment:
Somewhere at the start of the documentation it would be helpful to explain _how_ to run the commands that show up in code blocks throughout. Maybe something like:

Throughout this documentation you will see commands like
```
sudo apt update
```
These commands should be run in a terminal such as Bash (Linux/macOS), PowerShell, or Command Prompt (Windows).